Aesop/Search/Expansion/Simp.lean

```diff
@@ -44,12 +44,22 @@ def mkSimpOnly (stx : Syntax) (usedSimps : UsedSimps) (includeFVars : Bool) :
   let env ← getEnv
   for (thm, _) in usedSimps.toArray.qsort (·.2 < ·.2) do
     match thm with
-    | .decl declName inv => -- global definitions in the environment
```

Comment:
This function should actually be removed entirely and be replaced with core's `mkSimpOnly` (from `Lean.Elab.Tactic.Simp`).